addon/components/frost-toggle.js

```diff
@@ -53,45 +58,44 @@ export default Component.extend(SpreadMixin, PropTypeMixin, FrostEventsProxyMixi
   },
 
   getDefaultProps () {
+    const falseLabel = this.get('falseLabel')
+    const trueLabel = this.get('trueLabel')
+    const validTypes = ['string', 'number', 'boolean']
```

Comment:
I'm honestly not sure why we would care about the type of the label - I know this was there in the previous code, I just can't think of any reason for it